src/main/java/io/vertx/cassandra/ResultSet.java

```diff
@@ -60,22 +59,22 @@
    * @param handler handler called when one row is fetched
    * @see com.datastax.driver.core.ResultSet#one
    */
-  @GenIgnore
+  @SuppressWarnings("codegen-allow-any-java-type")
   @Fluent
   ResultSet one(Handler<AsyncResult<Row>> handler);
```

Comment:
The async result might be `null`, correct? In this case you should declare the method like this:

```java
ResultSet one(Handler<AsyncResult<@Nullable Row>> handler);
```

The difference is that the RxJava 2 generator will create a method returning `Maybe` instead of `Single` (which forbids `null` values.

Please verify other methods as well.

Reply:
> The async result might be null

Sure. Good point. Just marked with `@Nullable`.

I have also checked other methods, seems that this is the only method which asynchronously returns null.

Reply:
:+1: